tests/testqPrefDisplay.cpp

```diff
@@ -59,7 +60,7 @@ void TestQPrefDisplay::test_set_load_struct()
 	auto display = qPrefDisplay::instance();
 
 	display->set_display_invalid_dives(false);
-	display->set_divelist_font("helvitica");
+	display->set_divelist_font("Helvitica");
```

Comment:
That font is named Helv_e_tica , not Helv_i_tica
Applies everywhere here...

Reply:
You might have noted that travis was green, that is because it is just a string. Changed strings to secure nobody believes it is normal font names. However it is interesting to see what reviewers (@bstoeger made a similar comment) put weight on, to me the more interesting question is “why do the old code accept non existing fonts, and how to avoid it?” but is stuff for another PR.